### PR TITLE
Update Maven plugin doc and fix other docs typos

### PR DESCRIPTION
**Pull request by gwenneg**
Following #3024 merge, I made the same kind of changes in the Maven plugin doc.

### CODE REVIEW

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 1]
```diff
@@ -164,14 +164,28 @@ app.
...
+* `true` - the JVM will start in debug mode and be suspended until a debugger is attached to port `5005`
+* `client` - the JVM will start in client mode and attempt to connect to `localhost:5005`
+* `{port}` - the JVM will start in debug mode and be suspended until a debugger is attached to `{port}`
+
+So, you can run a Quarkus application in debug mode listening to port `5006` instead of the default port `5005` using:
 
 [source,shell]
 ----
-./mvnw compile quarkus:dev -Ddebug=true
+./mvnw compile quarkus:dev -Ddebug=5006
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]: This was an interesting example as it waits for you to attach your debugger.

Comment (gwenneg) [reply docs/src/main/asciidoc/maven-tooling.adoc]: The new example also waits for it as explained a few lines above. Do you specifically want the -Ddebug=true example in the doc?

Comment (gsmet) [reply docs/src/main/asciidoc/maven-tooling.adoc]: I think having `-Ddebug` as the first example with everything default would be nice as it's the one that people will use the most.

Defining another port is described above and I don't think it's the most common case.

**Comment by gwenneg** [comment]
You're right, I should have focused on what most users will need instead of that port changing case... :)

How about this new version? I'm not sure about making the `-Ddebug` a tip instead of an example because it sounds like both as the same time. At least, it goes straight to the most important point.

**Review by cescoffier — APPROVED** [review]

**Pull request merged**